core/client/hdfs/src/main/java/alluxio/hadoop/HadoopUtils.java

```diff
@@ -32,6 +34,18 @@
 @ThreadSafe
 public final class HadoopUtils {
   private static final Logger LOG = LoggerFactory.getLogger(HadoopUtils.class);
+  public static final String ZOOKEEPER_IDENTIFIER = "zk@";
```

Comment:
the "zk" and the "@" are logically two different parts of the authority, so it would be good to represent them that way.

Reply:
Could I change them into two parts? like ZOOKEEPER_IDENTIFIER = "zk" and ZK_SEPARATOR = "@" ?

Reply:
It will make it more difficult to use in our codes.